src/components/common/settings/SetText.tsx

```diff
@@ -55,7 +69,7 @@ const SetText: FC<Prop> = ({
       />
       <SaveButton
         buttonDataCy={buttonDataCy}
-        handleOnClick={() => handleClickSaveText(resourceKey, resourceText)}
+        handleOnClick={() => handleClickSaveText()}
```

Comment:
I think you can use it like this since you are not passing in any arguments.
```suggestion
        handleOnClick={handleClickSaveText}
```